week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? the math.random will get the  number and times it by 10 and then output the result
```

Comment:
What kind of a number will `Math.random()` get?

Reply:
@martynamaron, The Math.random() function returns a floating-point, pseudo-random number in the range 0–1